lib/fiet.ex

```diff
@@ -104,6 +104,7 @@ defmodule Fiet do
     |> case do
       {:ok, false} -> :error
       {:ok, type} -> type
+      _else -> :error
```

Comment:
I think it would be better to be explicit about what `_else` is, since `StackParser` only returns `{:ok, result} | {:error, _reason}`.

```suggestion
      {:error, _reason} -> :error
```